README.md

```diff
@@ -643,7 +643,8 @@ max_memory:
 # If you want to use 'lora' or 'qlora' or leave blank to train all parameters in original model
 adapter: lora
 # If you already have a lora model trained that you want to load, put that here.
-# This means after training, if you want to test the model, you should set this to the value of `lora_out_dir`.
+# This means after training, if you want to test the model, you should set this to the value of `output_dir`.
+# Note that if you merge an adapter to the base model, a new subdirectory `merged` will be created under the original directory, which you will want to use instead.
```

Comment:
```suggestion
# Note that if you merge an adapter to the base model, a new subdirectory `merged` will be created under the `output_dir`.
```

Reply:
Please also add info to find the merged model within `{output_dir}/merged` folder in the `### Merge LORA to base` section for visibility. I had some people ask where to find the final model.

Reply:
@NanoCode012 I have some questions about that section (`### Merge LORA ...`).  I'll message you on discord

Reply:
I'll merge this and then work on that section after I confirm it works like I think it does